content/src/fieldConfig/config.json

```diff
@@ -289,8 +289,8 @@
   },
   "footer": {
     "creditLinkText": "Open Source",
-    "officeLinkText": "Office of Innovation",
-    "officeTextTwo": " by the ",
+    "officeOfInnovationText": "Office of Innovation",
```

Comment:
[pebble] Rather than taking the text replacement step, I think it might be best to directly use the non-breaking space characters in this string. What do you think? I'm not clear on how the pieces fit together and I haven't tested it, but I'll defer to you if you had and the non-breaking spaces broke.